### PR TITLE
Nerfs trashbags. Can only hold trash now instead of being the ultimate toolbelt.

### PR DESCRIPTION
**Pull request by BlackMajor**
As a greyshit it hurts me to do this to my sweet child.

Balance PR so feel free to discuss. Namely on letting me know of any other "Trash" items this should probably carry.

-Trashbag size reduced to medium instead of bulky.
-Trashbag can_hold changed to only hold various "Trashy" items, including food, glass shards, paper, pens, circuits, sotck parts, syringes, bullets and of course. Trash.

Why?
It's probably not a good idea to have thirty grenades in your belt. Or any other combination of 30 small items.

### CODE REVIEW

**Comment by deathride58** [comment]
w h y  though. The entire point of the trash bag is that it's a belt slot backpack balanced via rarity in the form of exclusivity to the janitor (and the rest of service once bluespace bags are researched, at which point there's already plenty of options that outclass it)

File: code/game/objects/items/storage/bags.dm [hunk 1]
```diff
@@ -48,7 +48,7 @@
 	STR.max_w_class = WEIGHT_CLASS_SMALL
 	STR.max_combined_w_class = 30
 	STR.max_items = 30
-	STR.cant_hold = typecacheof(list(/obj/item/disk/nuclear))
+	STR.can_hold = typecacheof(list(/obj/item/ammo_casing, /obj/item/trash, /obj/item/dildo, /obj/item/electronics, /obj/item/circuitboard, /obj/item/reagent_containers/food, /obj/item/stock_parts, /obj/item/paper, /obj/item/reagent_containers/syringe, /obj/item/shard, /obj/item/pen))
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/storage/bags.dm]: i'd say pills and bottles/soda cans should be added in too as well as water bottles things like that...
Hard press to not add in cardboard, and metal rods. As those are commanly found around as a trash item, broken lights are also a thing unless thats under trash.

Comment (BlackMajor) [reply code/game/objects/items/storage/bags.dm]: Soda cans are a subtype of /food so they're in.

Cardboard, rods and broken lights are good ideas though thanks.

**Comment by BlackMajor** [comment]
> w h y though. The entire point of the trash bag is that it's a belt slot backpack balanced via rarity in the form of exclusivity to the janitor (and the rest of service once bluespace bags are researched, at which point there's already plenty of options that outclass it)

It's not janitor exclusive considering there are instances that exist in maint, space, and the service lathe being particularly public access (more so on some maps than others).

**Comment by EmeraldSundisk** [comment]
I can't think of EXACT items off the top of my head but what comes to mind is making the janitorial cart able to hold more janitor-relevant items to compensate.

**Comment by Owai-Seek** [comment]
Is it possible to blacklist troublesome items instead of making a whitelist for all the possible things a janitor would pick up in the line of duty?

Things I've picked up as a Janitor

- Service Borg Snacks (Cookies, Lolipops, Ice Cream)
- All the food/ingredients in the kitchen.
- Banana's and Banana Peels
- Junk Food and Junk Food Wrappers
- Soda Cans (and Crushed Soda Cans)
- All the bottles of alcohol in the vendor (broken and unbroken)
- Pipes
- Cables
- Floor Tiles
- Metal
- Glass
- Metal Rods
- (?) Wood (Not sure if that actually fits or not.)
- Glasses, Beakers, Bottles, Hypovials
- Toys
- Foam Force and Riot Foam Darts
- Random Medical Supplies
- Paper Cups
- Paper (and Paper Planes)
- Plates
- Sand
- Seeds
- Salt and Pepper Shakers
- Glass Shards
- Lights (and Broken Lights)
- All the different shell casings
- All Cigarette related items. (Matches, Cigarette butts, lighters, ecigs, ect.)
- Penlights
- Random Small stuff (Batteries, Igniters, Remotes)
- Destroyed Machines (and their components)
- Photos
- Glowsticks
- Decks of Cards
- Epipens
- Autosurgeons
- (?) Oxygen Tanks and Masks
- Clip Boards
- Stamps


There's probably a TON of things that I'm not remembering that could be placed here. When shit gets blown up a garbage bag is the -only- way to clean up (besides, I dunno, burning or spacing everything.) It shouldn't get a heavy-handed nerf when blacklisting items would work much better.

Just blacklist grenades and tools from the trash bag.

File: code/game/objects/items/storage/bags.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/storage/bags.dm]: bottles are not, as well as things like coffie and such...

**Comment by Tupinambis** [comment]
Just blacklist problem items instead of nerfing the functionality in general.

**Comment by KathrinBailey** [comment]
Blacklist tools, weapons and don't let ammo stack if it can stack.
Whitelisting means 10 more PRs for "adds obscure item to trashbag whitelist".

**Comment by BlackMajor** [comment]
Closing this in favour of Kevinz's smarter nerf
https://github.com/Citadel-Station-13/Citadel-Station-13/pull/9641

**Pull request closed without merging**